test/langtools/jdk/jshell/AbstractStopExecutionTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2015, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think we just need `2025,`, the beginning year of `2015` seems incorrect.

Reply:
Oops, thanks - cut & paste error. Fixed in 1603e17f31d.